python/ray/WebUI.ipynb

```diff
@@ -10,17 +10,266 @@
+    "qgrid.show_grid(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# System State Information \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Node Information"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "ctable = ray.global_state.client_table()\n",
+    "\n",
+    "client_list = []\n",
+    "for node_ip in ctable:\n",
+    "    for client in ctable[node_ip]:\n",
+    "        client[\"node_ip_address\"] = node_ip\n",
+    "        client_list.append(client)\n",
+    "\n",
+    "client_df = pd.DataFrame(client_list)\n",
+    "client_df.columns = [\"Aux Address\", \"Client Type\", \"DB Client ID\", \"Deleted\", \"Local Scheduler Socket\", \"Num CPUs\", \"NumGPUs\", \"Node IP Address\"]\n",
+    "qgrid.show_grid(client_df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Worker Information "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workers = rc.keys(\"Worker*\") \n",
+    "worker_info = dict()\n",
+    "for worker in workers:\n",
+    "    worker_key_str = worker[len('Workers:'):]\n",
+    "    worker_key_bytes = worker_key_str.encode('latin-1')\n",
+    "    worker_info['Workers:{}'.format(hex_identifier(worker_key_bytes))] = rc.hgetall(worker)\n",
+    "table = pd.DataFrame.from_dict(worker_info)\n",
+    "table_t = table.T\n",
+    "table_t.index.name = \"WorkerID\"\n",
+    "table_t.columns = [\"Local Scheduler Socket\", \"Node IP Address\", \"Plasma Manager Socket\", \"Plasma Store Socket\", \"Stderr File\", \"Stdout File\"]\n",
+    "qgrid.show_grid(table_t)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Object Store Information\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "object_dict = {oid.hex(): v for oid, v in ray.global_state.object_table().items()}\n",
+    "object_df = pd.DataFrame(object_dict).transpose()\n",
+    "object_df.index.name = \"ObjectID\"\n",
+    "object_df.columns = [\"Data Size\", \"Hash\", \"IsPut\", \"ManagerIDs\", \"TaskID\"]\n",
+    "qgrid.show_grid(object_df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Error Information\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "event_names = rc.keys(\"event_log*\")\n",
+    "error_profiles = dict()\n",
+    "for i in range(len(event_names)):\n",
+    "    event_list = rc.lrange(event_names[i], 0, -1)\n",
+    "    for event in event_list:\n",
+    "        event_dict = json.loads(event)\n",
```

Comment:
I guess this will change to just call `global_state.task_profiles()`, right? If not, we need to import `json` somewhere.